File: app/src/main/java/com/slimshady/bottomsheetmodule/util/ScreenUtils.java
package com.slimshady.bottomsheetmodule.util;

import android.app.Activity;
import android.content.Context;
import android.util.DisplayMetrics;

public class ScreenUtils {
    public static int getWindowHeight(Context context) {
        DisplayMetrics displayMetrics = new DisplayMetrics();
        ((Activity) context).getWindowManager().getDefaultDisplay().getMetrics(displayMetrics);
        return displayMetrics.heightPixels;
    }
}

File: app/src/main/java/com/slimshady/bottomsheetmodule/util/ScreenUtils.kt
package com.slimshady.bottomsheetmodule.util

import android.util.DisplayMetrics
import android.app.Activity
import android.content.Context

object ScreenUtils {
    @JvmStatic
    fun getWindowHeight(context: Context): Int {
        val displayMetrics = DisplayMetrics()
        (context as Activity).windowManager.defaultDisplay.getMetrics(displayMetrics)
        return displayMetrics.heightPixels
    }
}